charts/apisix/README.md

```diff
@@ -101,6 +101,18 @@ The command removes all the Kubernetes components associated with the chart and
 | customPlugins.plugins[0].configMap | object | `{"mounts":[{"key":"","path":""},{"key":"","path":""}],"name":""}` | plugin codes can be saved inside configmap object. |
 | customPlugins.plugins[0].configMap.mounts | list | `[{"key":"","path":""},{"key":"","path":""}]` | since keys in configmap is flat, mountPath allows to define the mount path, so that plugin codes can be mounted hierarchically. |
 | customPlugins.plugins[0].configMap.name | string | `""` | name of configmap. |
+| deployment.controlPlane | object | `{"cert":"","certKey":"","certsSecret":"","clientCACert":"","confServerPort":"9280"}` | used for control_plane deployment mode |
+| deployment.controlPlane.cert | string | `""` | conf Server CA cert name |
+| deployment.controlPlane.certKey | string | `""` | conf Server cert key name |
+| deployment.controlPlane.certsSecret | string | `""` | secret name used by conf Server |
+| deployment.controlPlane.clientCACert | string | `""` | conf Server mTLS cert name |
+| deployment.controlPlane.confServerPort | string | `"9280"` | conf Server address |
+| deployment.dataPlane | object | `{"controlPlane":{"host":[],"prefix":"/apisix","timeout":"31"}}` | used for data_plane deployment mode |
+| deployment.dataPlane.controlPlane.host | list | `[]` | The hosts of the control_plane used by the data_plane |
+| deployment.dataPlane.controlPlane.prefix | string | `"/apisix"` | The prefix of the control_plane used by the data_plane |
+| deployment.dataPlane.controlPlane.timeout | string | `"31"` | Timeout when the data plane connects to the control plane |
```

Comment:
The default timeout is 30? And it is not a string

Reply:
typo